### PR TITLE
Skeleton for LostPassword

### PR DESCRIPTION
**Pull request by pmattmann**
Only skeleton, because it will change with the form/service/validation concept again...

### CODE REVIEW

File: application/Core/Entity/Login.php [hunk 1]
```diff
@@ -115,7 +115,7 @@ public function getUser()
 	 */
 	public function createPwResetKey()
 	{
-		$this->pwResetKey = md5(unique(microtime(true)));
+		$this->pwResetKey = md5(microtime(true));
```
Comment (usu) [inline application/Core/Entity/Login.php]: http://blog.affien.com/archives/2006/12/03/md5microtime/

File: application/Module/WebApp/Controller/LoginController.php [hunk 2]
```diff
@@ -81,8 +100,72 @@ public function logoutAction()
...
+		{	
+			$passwordLost->populate($this->getRequest()->getParams());
+			
+			$resetKey = $this->loginService->forgotPassword($passwordLost->getValue('email'));
+			
+			/*
+			 * Todo: Remove postDispatch call together with the die() statement
+			 * 		 The postDispatch is only to call the Flash-Command on the EntityManager 
+			 */
+			$this->postDispatch();
```
Comment (usu) [inline application/Module/WebApp/Controller/LoginController.php]: I think we should start to work with the different environments. We have those hacks on several locations know. E.g.
development --> show key
stage, production --> send key by email

**Comment by usu** [comment]
A captcha for the password lost functionality would be nice

**Comment by pmattmann** [comment]
Thanks, I will work on all of these Issues... and after refactoring, it will be auto-mergable, again ;)

**Pull request closed without merging**